src/components/clusterConfiguration/discoveryTroubleshootingModal.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import {
+  Button,
+  Modal,
+  ButtonVariant,
+  ModalVariant,
+  TextContent,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
+import { ToolbarButton } from '../ui/Toolbar';
+import PrismCode from '../ui/PrismCode';
+
+type DiscoveryTroubleshootingModalButtonProps = React.ComponentProps<typeof Button> & {
+  ButtonComponent?: typeof Button | typeof ToolbarButton;
+  onClick?: () => void;
+};
+
+export const DiscoveryTroubleshootingModalButton: React.FC<DiscoveryTroubleshootingModalButtonProps> = ({
+  ButtonComponent = Button,
+  onClick,
+  children,
+  ...props
+}) => {
+  const [isModalOpen, setIsModalOpen] = React.useState(false);
+  const closeModal = () => setIsModalOpen(false);
+  const handleClick = onClick || (() => setIsModalOpen(true));
+  return (
+    <>
+      <ButtonComponent variant={ButtonVariant.link} {...props} onClick={handleClick} isInline>
+        {children}
+      </ButtonComponent>
+      <DiscoveryTroubleshootingModal isOpen={isModalOpen} onClose={closeModal} />
+    </>
+  );
+};
+
+type DiscoveryTroubleshootingModalProps = {
+  onClose: () => void;
+  isOpen: boolean;
+};
+
+export const DiscoveryTroubleshootingModal: React.FC<DiscoveryTroubleshootingModalProps> = ({
+  onClose,
+  isOpen,
+}) => {
+  return (
+    <Modal
+      title="Host Discovery Troubleshooting"
+      isOpen={isOpen}
+      actions={[
+        <Button key="close" variant={ButtonVariant.primary} onClick={onClose}>
+          Close
+        </Button>,
+      ]}
+      onClose={onClose}
+      variant={ModalVariant.large}
+    >
+      <TextContent>
+        <Text component={TextVariants.p}>
+          Booting up an ISO might take a few minutes, and might depend heavily on your hardware and
+          network configuration, however, if after a few minutes your machine does not show up in
```

Comment:
if you have verified that your host is on, DHCP is enabled and yet the host does not appear in the UI, you may try to login over SSH to the host in order to troubleshoot, for that you would need to know your machine IP-address (HOW?)

Reply:
done